src/libstd/io/mod.rs

```diff
@@ -764,6 +766,7 @@ pub trait Read {
     ///
     /// [`File`]s implement `Read`:
     ///
+    /// [`read`]: trait.Read.html#tymethod.read
```

Comment:
Use intra-doc links if possible, otherwise the links will sometimes be broken in `core`. In this case it wouldn't matter but intra-links are also shorter :)
```suggestion
    /// [`read`]: Read::read
```

Reply:
It compiles and points to the correct method, thanks for the catch 👍  !